src/app/pages/channels/details/channels.details.component.ts

```diff
@@ -74,59 +75,97 @@ export class ChannelsDetailsComponent implements OnInit {
     }
   }
 
-  onDisconnect(row: any) {
-    this.channelsService.disconnectThing(this.channel.id, row.id).subscribe(
-      resp => {
-        this.updateConnections();
-        this.notificationsService.success('Thing successfully disconnected', '');
-      },
-    );
+  onDisconnect() {
+    this.selectedToDisconn.forEach(thingID => {
```

Comment:
It's not really possible to do that since several requests are sent in a loop. This must be done in the backend and implement properly in the UI. I'll open an issue to don't forget

Reply:
Here is the issue: https://github.com/mainflux/mainflux/issues/1389

Reply:
Yes, but in the meantime you can make a service function that emulates multi-disconnect. However, proceed as you see fit. This is just a suggestion.